satpy/modifiers/filters.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for image filters."""
+import logging
+
+import xarray as xr
+from dask_image.ndfilters import median_filter
```

Comment:
By having the modifier defined in `visir.yaml` and having this import at import-time, you are essentially making `dask-image` a hard requirement of Satpy. I could be wrong...but I don't think I am.

Reply:
oh, indeed...